docker/Makefile

```diff
@@ -53,26 +53,16 @@ build: prep
 	docker buildx build \
 		${DOCKER_BUILD_FLAGS} \
 		--load \
```

Comment:
This is not quite right, because `--load` cannot be used in conjunction with `--platform` and multiple architectures as described in https://github.com/docker/buildx/issues/59. The local daemon doesn't support images for multiple architectures in this way.

I think we first have to push images to the local registry for all architectures, then load the appropriate one to be used with tests.

I'll tweak this a bit!